lib/webhooks/deployment.payload.json

```diff
@@ -5,6 +5,7 @@
     "sha": "69a8b72e2d3d955075d47f03d902929dcaf74033",
     "ref": "master",
     "task": "deploy",
+    "name": "baxterthehacker/public-repo",
```

Comment:
@pcasaretto: I checked our current production webhook payloads for deployment events, and I'm not seeing this attribute. Can you share a recent example that shows that this attribute is included in the payloads you're seeing?